runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -33,15 +33,21 @@
 
 extern bool isOrderedPair(uint8_t reloType);
 
-uint8_t *
-J9::AheadOfTimeCompile::emitClassChainOffset(uint8_t* cursor, TR_OpaqueClassBlock* classToRemember)
+uintptrj_t
+J9::AheadOfTimeCompile::getClassChainOffset(TR_OpaqueClassBlock *classToRemember)
```

Comment:
Please add a better commit message to explain why you're doing this refactoring. I expect that it's temporary, right? Since eventually all the calls to `emitClassChainOffset()` will be transitioned into calls to `getClassChainOffset()` right?

Reply:
Yup, eventually `emitClassChainOffset` will be removed.